00-Install_and_Setup/README.md

```diff
@@ -9,38 +16,76 @@ For the commands shown, `%` (and anything to the left of it) represents the
 terminal prompt. You do not need to copy it; instead only copy the command to the
 right of `%`.
 
-## 1. Clone This Repository
-
-Download the workshop folder using
-[git](https://help.github.com/articles/set-up-git/):
 
-    % git clone https://github.com/astropy/astropy-workshop.git
-
-If you don't have git installed, you can download the ZIP file by pressing the
-green *Clone or download* button at
-https://github.com/astropy/astropy-workshop and selecting *Download ZIP*.
-However, this option is not recommended because it impedes the ability to
-update your copy of the repository if updates are made.
-
-## 2. Install Miniconda (if needed)
+## 1. Install Miniconda (if needed)
 
 *Miniconda is a free minimal installer for conda. It is a small, bootstrap
 version of Anaconda that includes only conda, Python, the packages they depend
 on, and a small number of other useful packages, including pip, zlib and a few
 others. Note, though, that if you have either Miniconda or the full Anaconda
 already installed, you can skip to the next step.*
 
-Check if Miniconda is already installed.
+In a terminal window, check if Miniconda is already installed.
 
     % conda info
 
 If Miniconda is not already installed, follow these instructions for your
-operating system: https://docs.conda.io/en/latest/miniconda.html
+operating system: https://conda.io/projects/conda/en/latest/user-guide/install/index.html.
+Please be sure to install a **64-bit version** of Miniconda to ensure all packages work correctly.
+
 
 On Windows, you might also need
 [additional compilers](https://github.com/conda/conda-build/wiki/Windows-Compilers).
```

Comment:
They *shouldn't* need compilers unless we are having them build something from source (which we probably shouldn't for a workshop).

Reply:
Good point. The page that this links to looks intimidating. I think I should remove this item.